src/components/ProgrammeList/ProgrammeListItem.tsx

```diff
@@ -79,26 +73,7 @@ const ProgrammeListItem: FC<
           </Heading>
         </OakLink>
       </Flex>
-      <Box
-        $dropShadow={isHovered ? "subjectCardHover" : "subjectCard"}
-        $transition={"all 0.4s ease-out"}
-        $transform={isHovered ? "translateY(-8px)" : null}
-        $ph={16}
-      >
-        <Flex
-          $mt={2}
-          $mb={8}
-          $flexDirection={"row"}
-          $font={"body-3"}
-          $color={"oakGrey4"}
-        >
-          <Span $mb={4}>{`${unitCount} units`}</Span>
-          <Span $mh={4}>
-            <Icon name="dot" size={8} />
-          </Span>
-          <Span $font={"body-3"}>{`${lessonCount} lessons`}</Span>
-        </Flex>
-      </Box>
```

Comment:
has this info moved somewhere else or are we just removing it?

Reply:
we're just removing it